regression.py

```diff
@@ -131,11 +131,6 @@ def testADF_ADF2013_01_stopiter_MoOCl4_sp_adfout(logfile):
     # len(logfile.data.scfvalues[0]) == 11
     assert not hasattr(logfile.data, "scfvalues")
 
-def testADF_ADF2016_fa2_adf_out(logfile):

```

Comment:
I think you need to update your local git repository - you are deleting stuff I don't think you intended to :)